src/Security/InteractiveSecurityHelper.php

```diff
@@ -182,7 +184,36 @@ public function guessPasswordField(SymfonyStyle $io, string $userClass): string
         );
     }
 
-    public function getAuthenticatorClasses(array $firewallData): array
+    public function getAuthenticatorClasses(array $firewallData, bool $newAuthSystem): array
+    {
+        if (!$newAuthSystem) {
+            return $this->getGuardAuthenticatorClasses($firewallData);
+        }
+
+        $knownAuthenticators = [
+            'form_login' => FormLoginAuthenticator::class,
+            'json_login' => JsonLoginAuthenticator::class,
+        ];
```

Comment:
Obviously, this is incomplete. But I'm not sure if there is a better way of grabbing these. This seems very practical 👍